crates/swc/tests/tsc-references/emitExponentiationOperatorInTempalteString4ES6.2.minified.js

```diff
@@ -1,3 +1,3 @@
 //// [emitExponentiationOperatorInTempalteString4ES6.ts]
 var t1 = 10;
-++t1, ++t1, --t1, ++t1, ++t1, t1++, ++t1, --t1, --t1, ++t1, ++t1, --t1;
+++t1, t1++, --t1, ++t1, ++t1, t1++, t1++, --t1, --t1, ++t1, t1++, --t1;
```

Comment:
Is this change expected?

Reply:
No, but seems like it does no harm.

Reply:
This entire file should be elimintaed in future optimize.